mlir/test/Dialect/Rock/conv_to_gemm_layout_regularization.mlir

```diff
@@ -1,20 +1,20 @@
-// RUN: rocmlir-gen --arch gfx908 -p -operation conv2d -fil_layout gkyxc -in_layout ngchw -out_layout hwgkn \
+// RUN: rocmlir-gen --arch gfx908 -p -operation conv -fil_layout gkyxc -in_layout ngchw -out_layout hwgkn \
```

Comment:
Changed the operation but not the layout, but changed the expected layout keys in the transform maps.  A bit inconsistent.